src_new/adapters/prebid.js

```diff
@@ -492,13 +569,15 @@ function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, g
 			break;
 
 		case "pubmatic":
-	case "pubmatic2":
+		case "pubmatic2":
 			slotParams["publisherId"] = adapterConfig["publisherId"];
-		slotParams["adSlot"] = slotParams["slotName"] || generatedKey;
-			slotParams["wiid"] = impressionID;
-		slotParams["profId"] = adapterID == "pubmatic2"? adapterConfig["profileId"]: CONFIG.getProfileID();
+			slotParams["adSlot"] = slotParams["slotName"] || generatedKey;
+			if(isPrebidPubMaticAnalyticsEnabled === false){
+				slotParams["wiid"] = impressionID;
+			}
+			slotParams["profId"] = adapterID == "pubmatic2"? adapterConfig["profileId"]: CONFIG.getProfileID();
 			/* istanbul ignore else*/
-		if(adapterID != "pubmatic2" && window.PWT.udpv){
+			if(adapterID != "pubmatic2" && window.PWT.udpv){
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 2 tabs but found 3. (indent)](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15196)